experiments/AMIP/moist_mpi_earth/coupler_utils/flux_calculator.jl

```diff
@@ -47,11 +47,12 @@ function calculate_surface_fluxes_atmos_grid!(integrator, info_sfc)
     FT = eltype(integrator.u.c.ρ)
     thermo_params = CAP.thermodynamics_params(integrator.p.params)
     surface_flux_params = CAP.surface_fluxes_params(integrator.p.params)
+    uₕ_int = Geometry.UVVector.(Spaces.level(Y.c.uₕ, 1))
```

Comment:
Doesn't this allocate more than previously?

Reply:
Ill revert it back